waltz-ng/client/data-flow/components/data-flows-tabgroup/data-flows-tabgroup.js

```diff
@@ -139,7 +140,78 @@ function prepareGraphTweakers(dataFlowUtilityService,
 }
 
 
-function controller($scope, dataFlowUtilityService) {
+function mkEntityNameCell(entityDataObjectField, valueField, columnHeading, entityNavViewName) {
+    return {
```

Comment:
v. minor but perhaps make the nav view name the complete state name, e.g. `main.app.view`, rather than simply `app`.  Would make the param list clearer later on..... ?